app/backends.py

```diff
@@ -19,32 +19,34 @@ def configure_user(self, user: User) -> User:
 
         :returns: [User] The user object with updates to is_staff, email, first_name, last_name fields.
         """
-        url = "https://yalies.io/api/people"
-        querystring = {"query": user.username}
+        url = 'https://yalies.io/api/people'
+        payload = {
+            'filters': {
+                'netid': user.username,
+            }
+        }
         headers = {
-            'Authorization': f"Bearer {get_secret('YALIES_API')}",
+            'Authorization': 'Bearer ' + get_secret('YALIES_API'),
         }
 
         try:
-            response = requests.post(url, headers=headers, json=querystring)
+            response = requests.post(url, headers=headers, json=payload)
```

Comment:
Figured it was ok to rename from querystring to payload because this is not a query string anymore